src/htmltojsx.js

```diff
@@ -468,6 +481,12 @@ StyleParser.prototype = {
       var key = style.substr(0, firstColon);
       var value = style.substr(firstColon + 1).trim();
       if (key !== '') {
+        // Lowercase style name if not vendor specific
+        // TODO better vendor prefix handling
+        if(key[0] != '-') {
```

Comment:
Might be cleaner to use [`charAt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charAt) since you know it's a String.